cfg/eks-1.7.0/controlplane.yaml

```diff
@@ -9,8 +9,7 @@ groups:
     text: "Logging"
     checks:
       - id: 2.1.1
-        text: "Enable audit Logs (Manual)"
-        type: manual
+        text: "Enable audit Logs (Automated)"
```

Comment:
Either add audit section OR keep the type as manual because there is no audit yet. Text can be changed as upstream has it as automated.